src/main.py

```diff
@@ -210,7 +213,7 @@ def __call__(self, parser, namespace, values, option_string=None):
 matplotlib {mpl}
 """.format(py=py_version, np=numpy.__version__, ne=numexpr.__version__,
            pt=tables.__version__, vt=vt_version, mpl=mpl_version,
-           bc=bcolz.__version__, yml=yaml.__version__))
+           bc=bcolz.__version__ if bcolz is not None else None, yml=yaml.__version__))

```

Comment:
plz use N/A when missing, and use the same template (bc_version = ...) than for other optional deps (it is not especially better, but lets be consistent)